appengine/endpoints/app.js

```diff
@@ -15,15 +15,12 @@
 
 'use strict';
 
-// [START app]
-// [START setup]
 const express = require('express');
 const bodyParser = require('body-parser');
 const Buffer = require('safe-buffer').Buffer;
 
 const app = express();
 app.use(bodyParser.json());
-// [END setup]
 
 app.post('/echo', (req, res) => {
```

Comment:
Is this sample not used in the docs anymore? If so, can/should we delete it outright?